airbyte-integrations/connectors/source-microsoft-teams/source_microsoft_teams/schemas/team_device_usage_report.json

```diff
@@ -3,40 +3,53 @@
   "type": ["null", "object"],
   "additionalProperties": true,
   "properties": {
-    "report_refresh_date": {
+    "reportRefreshDate": {
+      "type": ["null", "string"],
+      "format": "date"
+    },
+    "userId": {
+      "type": ["null", "string"]
+    },
+    "userPrincipalName": {
+      "type": ["null", "string"]
+    },
+    "lastActivityDate": {
+      "type": ["null", "string"]
+    },
+    "isDeleted": {
       "type": ["null", "string"]
     },
-    "user_principal_name": {
+    "deletedDate": {
       "type": ["null", "string"]
```

Comment:
Did not add a "date" format to this field as it defaults to an empty string when empty rather than a null value, which will throw a validation error.